### PR TITLE
Add eslint-plugin-mozilla to eslintrc config

### LINKED ISSUE #864: Add eslint-plugin-mozilla?
In https://github.com/mozilla-services/screenshots/pull/2700 @Standard8 added the eslint-plugin-mozilla plugin so we could easily merge into moz-central (ref: https://github.com/mozilla-services/screenshots/issues/2365).

Not sure if we want to do something similar here, or if having matching code is important to us.

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #864

### CODE REVIEW

File: components/player-view.js [hunk 1]
```diff
@@ -135,7 +135,7 @@ module.exports = class Player extends React.Component {
     window.AppData.set({loaded: true, exited: false});
     if (this.props.queue[0].live) this.setState({time: 'LIVE'});
     if (duration) {
-      window.AppData.set({duration: duration});
+      window.AppData.set({duration});
```
Comment (pdehaan) [inline components/player-view.js]: Yeah, https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand
And the ESLint rule is http://eslint.org/docs/rules/object-shorthand

Pretty sweet.

File: lib/get-random-id.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 module.exports = function() {
-  return require('sdk/util/uuid').uuid().number.replace('{','').replace('}','');
+  return require('sdk/util/uuid').uuid().number.replace('{', '').replace('}', '');
```
Comment (pdehaan) [inline lib/get-random-id.js]: I imagine so. The added spaces are just around the comma formatting, not around any strings or anything particularly interesting/useful.

**Comment by pdehaan** [comment]
That's a most unexpected Travis-CI failure.
Not sure what that's about. ESLint and prepublished both worked for me locally.

**Comment by Standard8** [comment]
> That's a most unexpected Travis-CI failure.
> Not sure what that's about. ESLint and prepublished both worked for me locally.

eslint-plugin-mozilla needs v6.9.1 or later, so that it has the "includes" functionality.

**Comment by meandavejustice** [comment]
@pdehaan Happy to merge once travis is passing

**Comment by pdehaan** [comment]
@meandavejustice Any objections to me bumping major Node version to v6?

**Comment by meandavejustice** [comment]
@pdehaan so long as it passes tests I don't mind :)

**Comment by pdehaan** [comment]
@meandavejustice Done!

**Pull request merged**